pkg/storaged/client.js

```diff
@@ -206,6 +214,203 @@
     return false;
 }
 
+export async function btrfs_poll() {
+    const usage_regex = /used\s+(?<used>\d+)\s+path\s+(?<device>[\w/]+)/;
+    if (!client.uuids_btrfs_subvols)
+        client.uuids_btrfs_subvols = { };
+    if (!client.uuids_btrfs_usage)
+        client.uuids_btrfs_usage = { };
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19690-20240108-153441-e8766e70-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)